src/qdrant.rs

```diff
@@ -264,6 +271,8 @@ pub mod quantization_config {
         Scalar(super::ScalarQuantization),
         #[prost(message, tag = "2")]
         Product(super::ProductQuantization),
+        #[prost(message, tag = "3")]
+        Binary(super::BinaryQuantization),
```

Comment:
Probably too late for this but why is BinaryQuantization not a variant of ScalarQuantization?

Reply:
Answer by @IvanPleshkov in chat:
> Usage of binary quantization is different. In regular scalar quantization you can use any data with any metric. On another hand, binary quantization has strong limitations. Values should be in range [-1; 1] and for now it is applicable for openai embeddings only
> Also we are thinking about vectors transforming for binary quantization. I mean conversion from N dimensional vectors into M * N vectors which should be binary-quantization-friendly. If we do that, it will be options related to binary case only